specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.5/TextAnalytics.json

```diff
@@ -894,8 +894,8 @@
               "type": "string"
             },
             "loggingOptOut": {
-              "default": "true",
-              "type": "string"
+              "default": true,
+              "type": "boolean"
```

Comment:
The change to `"default"` makes the code generator not set the default value correctly.

Reply:
changed it back